### DISCUSSION TITLE
Wled hardware used with bambulab

### DISCUSSION THREAD

**Discussion by filipkotian** [greghesp/ha-bambulab#269] [Q&A]
Hi @greghesp!

Many thanks for this integration, I like it a lot! I found out in your description, that it is possible to use Wled with bambulab x1c. Are you currently using it? Could you share with me hardware you used? I mean controller, Led strips and also how it is powered. Many thanks!

Filip

**Comment by greghesp** [comment]
Hey - I didn't actually add this one in.  @dreed47 might be able to give you some insight
